packages/@lwc/engine-core/src/framework/reporting.ts

```diff
@@ -47,6 +48,7 @@ export type ReportingPayloadMapping = {
     [ReportingEventId.NonStandardAriaReflection]: NonStandardAriaReflectionPayload;
     [ReportingEventId.TemplateMutation]: TemplateMutationPayload;
     [ReportingEventId.StylesheetMutation]: StylesheetMutationPayload;
+    [ReportingEventId.UnexpectedStylesheetContent]: BasePayload;
```

Comment:
I originally had a payload interface for this, but I don't see a point since we don't need any properties beyond what's provided by `BasePayload`. We can always add a new payload interface if that ever changes.